packages/dapp/src/hooks/userTracking/useUserTracking.ts

```diff
@@ -114,14 +109,17 @@ export function useUserTracking() {
         });
       }
       if (
-        !disableTrackingTool?.includes(EventTrackingTool.Raleon) &&
-        account.isActive &&
-        account.address
+        !disableTrackingTool?.includes(EventTrackingTool.Cookie3) &&
+        cookie3?.trackEvent({
+          category,
+          action,
+          name: label,
+          //value: todo add to trackingEventProps
+        })
       ) {
-        window.raleon.registerEvent(action, account.address, category);
       }
     },
-    [account.address, account.isActive, arcx],
+    [arcx, cookie3],
```

Comment:
why is this not dependant on the account data anymore?

Reply:
We had used the `account.address` and `account.isActive` only for Raleon and don´t need it for cookie3:

**This was before:**
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/39acc602-0236-4286-9b61-51079ffa8e28)